### ISSUE TITLE
Font size for tabs should be customizable

### ISSUE THREAD

**Issue by da2x** [brave/browser-laptop#4650]
The text labels on tabs is now so small that it isn’t legible anymore. Wasn’t a problem in earlier versions that used a larger text size. The text on the title bar has also shrunk, but the text there is still legible.

![tiny-tab-text](https://cloud.githubusercontent.com/assets/1102886/19241687/0baf1f62-8f10-11e6-887d-5f6e15e8a703.png)

The font-size now is 8-9px, and it used to be something like 11-12px. Causes strain on the eyes. This is an accessibility issue.

Introduced somewhere between v0.12.3 dev and 0.12.4 dev.

Linux, Fedora 24, 64-bit, GNOME desktop, not a HIDPI issue. Not a problem on macOS 10.10 where the tabs use a bigger font size.

**Comment by bsclifton** [comment]
Was updated w/ https://github.com/brave/browser-laptop/pull/4400

cc: @bradleyrichter

**Comment by bradleyrichter** [comment]
AFAIK, this is a bug caused by the font change, not a size change. Can we revert or bring back the Ariel fallback in the right place within the font definition to allow Linux to behave correctly?

**Comment by Aeyoun** [comment]
For reference, the Linux desktop font stack these days is:
* Ubuntu (for Unity desktop)
* Cantarell (for GNOME desktop)
* Oxygen-Sans (for KDE desktop)
* Noto Sans (for Mate desktop)

`Ubuntu,Cantarell,Oxygen-Sans,'Noto Sans';`

The fonts in that order is also a good CSS font stack as the list’s order roughly matches the various desktop environment’s market shares. (No lynch mob, please.)

**Comment by bsclifton** [comment]
Updated issue title to have an action which can be acted on 😄

**Comment by luixxiul** [comment]
@bsclifton do you think this issue can be closed thanks to your implementation on UI resizing?

**Comment by bradleyrichter** [comment]
no because the default may never change for a user who isn't hunting through prefs. We may need to add linux-specific CSS if this is not actually a bug.

**Comment by christianbundy** [comment]
Just installed Brave for the first time and the unreadable tabs was the first thing I noticed.

![screenshot from 2018-04-23 11-42-13](https://user-images.githubusercontent.com/537700/39146354-6702fc00-46eb-11e8-8e75-fddeb6605a36.png)

The body text is set to 16pt (Nimbus Sans), whereas the UI elements seem to have much a much smaller size. It seems like changing the UI scale to "Large" upgrades them to a reasonable and accessible size, but if this is only happening on Linux then this seems like a bug.

![screenshot from 2018-04-23 11-45-42](https://user-images.githubusercontent.com/537700/39146686-58db2066-46ec-11e8-8e14-d5e7c29f9613.png)

```console
$ uname -a
Linux samus 4.16.2-3-ph #1 SMP Mon Apr 16 15:17:56 PDT 2018 x86_64 GNU/Linux
```

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/774
